docs/pages/basics/quick-start-guide/index.md

```diff
@@ -8,6 +8,7 @@ Follow these steps to get started with Storybook.
 
 ```sh
 npm i -g @storybook/cli
+cd my-project-directory
```

Comment:
On the other hand, we can assume that the user is already in the project folder, so it might be worth to put this line first